index.html

```diff
@@ -12,10 +12,71 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="header"> <!-- contains logo and nav -->
+        <div class="container">   
+            <div class="header__inner">
+               <img src="img/karma-logo.svg" width="25" alt="karma-logo">
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+                <nav class="nav">
+                    <a  href="#">Meet Karma</a>
```

Comment:
Good use of the nav tag. Could you improve this to include an unordered list and list items?